packages/web/src/js/FileUploader.ts

```diff
@@ -224,8 +251,12 @@ class FileUploader extends BaseComponent {
       this.checkAllowedFileType(file);
       this.checkQueueLimit();
       this.appendToList(file);
+      this.updateDropZoneVisibility();
+      EventHandler.trigger(this.wrapper, EVENT_CHANGE, { fileQueue: this.fileQueue });
     } catch (error) {
+      // eslint-disable-next-line no-console
       console.warn(error);
```

Comment:
Now I am not sure if we should log this into the browser console. We cannot turn this off on production and also we are adding the error to the event, which can be handled on demand by the developer.

So I suggest removing the console.warn.

Reply:
done